src/modules/export/components/dialog/ExportDialogContent.js

```diff
@@ -19,8 +20,10 @@ const Update_Media_Related_Properties = 'update_isPortrait';
  * @author thiloSchlemmer
  */
 export class ExportDialogContent extends MvuElement {
+	#exportTypes;
 	constructor() {
 		super({ exportData: null, isPortrait: false });
+		this.#exportTypes = this._getExportTypes();
```

Comment:
Can't see a reason for this field.  `_getExportTypes()` could be called within `createView` without any difficulty.